### DISCUSSION TITLE
Issue while calculating ROC

### DISCUSSION THREAD

**Discussion by shivam221098** [moj-analytical-services/splink#2042] [Q&A]
When I try to get the roc curve using the `roc_chart_from_labels_column` method, I get this error. Don't know how can I solve this as I don't have any of my methods used in the code. Just created a model using

```python
spark_df = spark.read.csv("gs://temp_csv.csv", header=True)
linker = SparkLinker(spark_df, settings, spark=spark)
    
# probablity two random records match
linker.estimate_probability_two_random_records_match(deterministic_rules, recall=0.8)

linker.estimate_u_using_random_sampling(max_pairs=1e7)  # u estimation

linker.estimate_m_from_label_column("column_label")
```

After above, I just save the model into dictionary and when I apply a method, I get this error.

![image](https://github.com/moj-analytical-services/splink/assets/56783600/6561f926-75d6-4810-8b62-5220d2af9f89)

**Comment by RobinL** [comment]
Thanks for the report - I think this is likely a bug.  I'm going to convert into an issue 

Issue is [here](https://github.com/moj-analytical-services/splink/issues/2043).  I've closed it - see issue

**Reply by shivam221098** [reply]
Sure. Thanks.
